conda_build/environ.py

```diff
@@ -47,6 +47,10 @@ def get_git_build_info(src_dir, git_url, expected_rev):
     env = os.environ.copy()
     d = {}
     git_dir = join(src_dir, '.git')
+    if not isinstance(git_dir, str):
```

Comment:
In python 2 `git_dir` will be `unicode`.  In python 3, it will be `str` (which is natively unicode, but that's fine -- in python 3 `subprocess` handles `str`).

So, I want this behavior:

```
if isinstance(git_dir, unicode):
```

But `unicode` isn't even defined as a keyword in python 3, so I can't really use that word, so I check if it's *not* a string.  Easy enough.